api/v1/ytsaurus_types.go

```diff
@@ -618,10 +622,33 @@ type TabletCellBundleInfo struct {
 	TabletCellCount int    `yson:"tablet_cell_count,attr" json:"tabletCellCount"`
 }
 
+type UpdateSelector string
+
+const (
+	UpdateSelectorNone            UpdateSelector = ""
+	UpdateSelectorNothing         UpdateSelector = "Nothing"
+	UpdateSelectorStatelessOnly   UpdateSelector = "StatelessOnly"
+	UpdateSelectorMasterOnly      UpdateSelector = "MasterOnly"
+	UpdateSelectorTabletNodesOnly UpdateSelector = "TabletNodesOnly"
+	UpdateSelectorExecNodesOnly   UpdateSelector = "ExecNodesOnly"
+	UpdateSelectorEverything      UpdateSelector = "Everything"
+)
+
+type UpdateFlow string
+
+const (
+	UpdateFlowNone        UpdateFlow = ""
+	UpdateFlowStateless   UpdateFlow = "Stateless"
+	UpdateFlowMaster      UpdateFlow = "Master"
+	UpdateFlowTabletNodes UpdateFlow = "TabletNodes"
+	UpdateFlowFull        UpdateFlow = "Full"
+)
+
 type UpdateStatus struct {
 	//+kubebuilder:default:=None
 	State                 UpdateState            `json:"state,omitempty"`
 	Components            []string               `json:"components,omitempty"`
+	Flow                  UpdateFlow             `json:"flow,omitempty"`
```

Comment:
Please add comment that this field means - it goes into CRD and api.md

Reply:
done